src/main/java/by/a1qa/screens/TicketScreen.java

```diff
@@ -0,0 +1,49 @@
+package by.a1qa.screens;
+
+
+import aquality.appium.mobile.elements.interfaces.IButton;
+import aquality.appium.mobile.elements.interfaces.ILabel;
+import aquality.appium.mobile.screens.Screen;
+import org.openqa.selenium.By;
+
+
+public class TicketScreen extends Screen {
+    private final ILabel seatingChartLoc = getElementFactory().getLabel(By.id("com.todaytix.TodayTix:id/static_seating_chart"), "Seating chart");
+    private final ILabel expensiveTicket = getElementFactory().getLabel(By.id("com.todaytix.TodayTix:id/real_price"), "Expensive ticket");
+    private final ILabel holdTimerLbl = getElementFactory().getLabel(By.id("com.todaytix.TodayTix:id/hold_timer_static_label"), "Hold timer");
+    private final ILabel numberAndPriceLbl = getElementFactory().getLabel(By.id("com.todaytix.TodayTix:id/price_label"), "Nmber and price of ticket");
+    private final IButton continueButton = getElementFactory().getButton(By.id("com.todaytix.TodayTix:id/checkout_button"), "Continue butotn");
+
+    public TicketScreen() {
+        super(By.id("com.todaytix.TodayTix:id/static_seating_chart"), "Ticket Screen");
+    }
+
+    public boolean isOnTicketScreen() {
+        return seatingChartLoc.state().waitForDisplayed();
+    }
+
+    public boolean waitForHoldTimer() {
+        return holdTimerLbl.state().waitForDisplayed();
+    }
+
+    public void clickMostExpensiveTicket() {
+        expensiveTicket.click();
+        expensiveTicket.click();
```

Comment:
2 times?

Reply:
Yeah, because 1 time isnt working well(Worked for 1 of 20 tests)

Reply:
then you need to think about retrying mechanism. Having just two same clicks is unacceptable.